js-rattler/Cargo.toml

```diff
@@ -42,7 +42,7 @@ rattler_solve = { path = "../crates/rattler_solve", default-features = false, fe
 url = "2.5.4"
 
 chrono = { version = "0.4.40", features = ["wasmbind"] }
-reqwest = "0.12.24"
+reqwest = { version = "0.12.24", default-features = false }
 reqwest-middleware = "0.4.2"
```

Comment:
same for this one probably wrt `default-features`. Does it then remove openssl from the lockfile again when re-locking?